docker/doni/doni-base/Dockerfile.j2

```diff
@@ -7,40 +7,19 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}"
 
 {{ macros.configure_user(name='doni') }}
 
-{% if install_type == 'binary' %}
-    {% if base_package_type == 'rpm' %}
-        {% set doni_base_packages = ['openstack-doni-common'] %}
-    {% elif base_package_type == 'deb' %}
-        {% set doni_base_packages = ['doni-common'] %}
-    {% endif %}
+{% if base_package_type == 'rpm' %}
+    {% set doni_base_packages = ['openstack-doni-common'] %}
+{% elif base_package_type == 'deb' %}
+    {% set doni_base_packages = ['doni-common'] %}
+{% endif %}
 
 {{ macros.install_packages(doni_base_packages | customizable("packages")) }}
 
-{% elif install_type == 'source' %}
-
-ADD doni-base-archive /doni-base-source
-
-{% set doni_base_pip_packages = [
-    '/doni'
-] %}
-
-RUN ln -s doni-base-source/* doni \
-    && {{ macros.install_pip(doni_base_pip_packages | customizable("pip_packages")) }} \
-    && mkdir -p /etc/doni \
-    && cp -r /doni/etc/doni/* /etc/doni/ \
-    && chown -R doni: /etc/doni \
-    && sed -i 's|^exec_dirs.*|exec_dirs=/var/lib/kolla/venv/bin,/sbin,/usr/sbin,/bin,/usr/bin,/usr/local/bin,/usr/local/sbin|g' /etc/doni/rootwrap.conf
-
-ADD doni_sudoers /etc/sudoers.d/kolla_doni_sudoers
-RUN chmod 750 /etc/sudoers.d \
-    && chmod 440 /etc/sudoers.d/kolla_doni_sudoers
```

Comment:
I think what we need to include is this section, and exclude the "binary" section.
Since we don't publish deb or rpm packages for doni, installing via deb or RPM won't work.